test/correctness/mul_div_mod.cpp

```diff
@@ -5,6 +5,7 @@
 #include <algorithm>
 
 using namespace Halide;
+using namespace Halide::Internal;
```

Comment:
Is it possible to tighten this down to specific symbols without adding too many? I don't have a strict objection to doing this, but narrowing use of Internal both prevents breakages due to coincidental name conflicts and signals to others that this is not a technique to copy.